crates/libtiny_tui/src/msg_area/mod.rs

```diff
@@ -128,12 +133,13 @@ impl MsgArea {
 // Scrolling
 
 impl MsgArea {
-    fn lines_height(&mut self) -> i32 {
+    /// The total number of visible lines if each Line was rendered at the current screen width
+    fn total_visible_lines(&mut self) -> i32 {
```

Comment:
I found this method a bit confusing. I think we need different naming for the messages added to the messaging area and lines shown in TUI. Currently I think we call both "lines". Perhaps we could call the former "message" and the latter "line". Messages do not change as we resize, but lines do. Then name of this functions becomes less confusing. It would look like this:

```rust
/// Total number of visible lines when each `Message` rendered at the current messaging area width
fn total_visible_lines(...) { ... }
```